js/services/contact_service.js

```diff
@@ -93,6 +93,9 @@ angular.module('contactsApp')
 		newContact.uid(newUid);
 		newContact.setUrl(addressBook, newUid);
 		newContact.addressBookId = addressBook.displayName;
+		if (_.isUndefined(newContact.fullName()) || newContact.fullName() === '') {
+			newContact.fullName(t('contacts', 'New Contact'));
```

Comment:
So it should be `New.Contact.fullName` everywhere 🤔 ... this would also affect lines which are not touched by this PR - so I think I am misunderstanding you @jancborchardt